docs/modules/ROOT/pages/faqs.adoc

```diff
@@ -1,83 +1,100 @@
 
-=== Is DataStax Cassandra Data Connector an open-source project?
+=== Is DataStax Cassandra Source Connector an open-source project?
 
-Yes, DataStax Cassandra Data Connector is open source. See the <<gitHubRepos,repos FAQ>>.
+Yes, DataStax Cassandra Source Connector is open source under the Apache 2.0 License. See the <<gitHubRepos,repos FAQ>>.
 
-=== Which Kubernetes platforms are supported by DataStax Cassandra Data Connector?
+=== Which Kubernetes platforms are supported by the CSC?
 
-They include Minikube, K8d, Kind, Google Kubernetes Engine (GKE), Microsoft Azure Kubernetes Service, Amazon Kubernetes Service (AKS), and other commonly used platforms.
+The CSC supports most commonly used Kubernetes platforms, including:
+
+* Minikube
+* K8d
+* Kind
+* Google Kubernetes Engine (GKE)
+* Microsoft Azure Kubernetes Service
+* Amazon Kubernetes Service (AKS), and other commonly used platforms.
 
 [#gitHubRepos]
-=== Where is the DataStax Cassandra Data Connector public GitHub repo?
+=== Where is the CSC public GitHub repo?
 
-The source for this FAQs document is co-located with the Cassandra Data Connector repo code, examples, and additional documentation: access the [repo here](https://github.com/riptano/cdcrep).
+The source for this FAQ and related CSC documents are co-located with the Cassandra Source Connector repo code, examples, and additional documentation: access the [repo here](https://github.com/riptano/cdcrep).
 
 == Installation
 
-=== Is there a prerequisite version of Java needed for the DataStax Cassandra Data Connector installation?
+=== Is there a prerequisite version of Java needed for the DataStax Cassandra Source Connector installation?
+
+The DataStax Cassandra Source Connector distribution is designed for Java 11. However, because the product releases Docker images, you do not need to install Java (8 or 11) in advance. Java 11 is bundled in the Docker image.
+
+=== How do I install Cassandra Source Connector?
 
-The DataStax Cassandra Data Connector distribution is designed for Java 11. However, because the product releases Docker images, you do not need to install Java (8 or 11) in advance. Java 11 is bundled in the Docker image.
+Download the tar file from the DataStax [Downloads site](https://downloads.datastax.com/#csc). Then follow the xref:install.adoc[install] instructions. The tarball contains the Cassandra Source Connector distribution's NAR file.
 
-=== How do I install Cassandra Data Connector?
+=== How many Cassandra Tables can I track?
 
-Download the tar file from the DataStax [Downloads site](https://downloads.datastax.com/#cdc). Then follow the xref:install.adoc[install] instructions. The tarball contains the Cassandra Data Connector distribution's NAR file.
+As many as your system can handle.  Note that each Cassandra table CDC events must be tracked with a separate Cassandra Source Connector.
```

Comment:
What is required for a system to be able to handle the tracking?

Is CDC correct here?

Reply:
CDC is correct - we're capturing the Cassandra table CDC events, and pushing them to Pulsar.

I'll leave a note here to ask about the system configuration.

@vroyer: Similar to the question above, what's the recommended hardware requirements?